src/components/Schedule/style.css

```diff
@@ -0,0 +1,21 @@
+#calendar{
+    display:flex;
+    justify-content: center;
+    margin: 0;
+}
+
+#schedule{
+    padding: 181px 0 181px 0;
+    background: url("../../assets/wavy-background.svg");
+    background-size: cover;
+}
+
+#text{
+    margin: 0 0 87px 120px;
+}
+
+h1{
```

Comment:
oops another small change here i just noticed: Can we use a class name or id selector for this header so it doesn't conflict with other sections.